### ISSUE TITLE
Explore alternate Push interfaces

### ISSUE THREAD

**Issue by jeffpaul** [10up/distributor#396]
**Is your enhancement related to a problem? Please describe.**
Documenting this exploration as an issue to consider/discuss having Push UI outside of the Admin Bar (e.g. Gutenberg-specific flows, Classic Editor-specific flows).  How prominent is Distribution from the front-end?  Following Gutenberg patterns, document level actions are in the toolbar?  Pros and cons of Admin Bar versus Gutenberg/Classic Editor flows (or instead of "versus", "along with").

**Describe the solution you'd like**
- [ ] display available state with usable connections
- [ ] add links to help doc, specifically the "Distributing content" section
- [ ] add `Success` state after submitting Push requests
- [ ] add `Distributed` state to see where content has been distributed to (e.g. to understand impacted sites if content is edited)
- [ ] add state to note when there are no connections set up yet / available to distribute to

**Designs**
Where should the distribute action be, to be a bit more discoverable and consistent with other doc level actions?
1. https://cldup.com/vbYyZ2JAuy.png
1. https://cldup.com/wzQa9foZ9E.png

**Describe alternatives you've considered**
A more consolidated distribution flow including an idea to create a consistent and more permanent home for ALL Distributor actions:
1. https://cldup.com/xCFW9FdoS7.png
1. https://cldup.com/yNFZZmxzhb.png
1. https://cldup.com/XH7qarZ-Uv.png
1. https://cldup.com/cbeYeLEsH8.png

**Additional context**
Relates to #116, #141, #171, #275, #374.

Props GH:@oszkarnagy / WP:[@oszkarnagy](https://profiles.wordpress.org/oszkarnagy/) for the UX analysis and recommendation.

**Comment by jeffpaul** [comment]
**TLDR**: we considered this, but have decided against acting on this issue.

**In more detail...**
@jakemgold, @helen, and I discussed this and while we can see where some people would like an alternate Push interface, we feel that the existing Admin Bar-based Push option works well as-is and that we would be adding unnecessary complexity in having to support multiple Push interfaces.  Another concern with integrating deeper into Gutenberg is the pace of changes in the Gutenberg codebase and that we might have to spend an inordinate amount of time updating our Push interface as Gutenberg updates and changes (whereas the Admin Bar is relatively stable and unchanging).  We believe a possible root cause behind this request is a lack of awareness where the Push interface is, so we've opened issues #408 and #409 to help with better education in our docs and callouts in WP Admin to better support the new user experience.
